### PR TITLE
Fixing loerm in about

### PR DESCRIPTION
**Pull request by sarakahaveci**
PTAL

### CODE REVIEW

File: src/components/about/AboutHeader.jsx [hunk 1]
```diff
@@ -11,8 +11,9 @@ export default function AboutHeader() {
       <div className="aboutLine"></div>
       <h1>Who are we</h1>
       <p className="aboutDescription">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua.
+        We are a group of four people who aims to enhance the health of everyone{" "}
+        <br />
+        by both strengthen the read about health and providing healthy products.
       </p>
```
Comment (emrerdem1) [inline src/components/about/AboutHeader.jsx:18]: Can you change this sentence with the one I texted?

File: src/components/home/Categories.jsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ const CATEGORIES_INFO = [
     img: sterhoscopeImg,
   },
   {
-    name: "Nutrition",
+    name: "NUTRITION",
     img: NutritionImg,
```
Comment (emrerdem1) [inline src/components/home/Categories.jsx]: Instead of changing it here manually, can you please use `text-transform: uppercase` for that className?

**Review by louisrli — APPROVED** [review]

**Comment by louisrli** [comment]
pls make emres change

File: src/components/about/AboutHeader.jsx (hunk 1 above)
Comment (sarakahaveci) [reply src/components/about/AboutHeader.jsx:18]: sure i did!

File: src/components/home/Categories.jsx (hunk 2 above)
Comment (yamoonz) [reply src/components/home/Categories.jsx:68]: there is no className for this one?

Comment (sarakahaveci) [reply src/components/home/Categories.jsx]: done!

**Comment by sarakahaveci** [comment]
> pls make emres change

sure i did it :)

**Review by emrerdem1 — APPROVED** [review]
Merge the latest changes and you are good to go! 💯

**Pull request merged**